tests/lib/rules/no-var.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @fileoverview Tests for no-var rule.
+ * @author Jamund Ferguson
+ */
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var eslint = require("../../../lib/eslint"),
+    ESLintTester = require("eslint-tester");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+var eslintTester = new ESLintTester(eslint);
+eslintTester.addRuleTest("lib/rules/generator-star", {
+    valid: [
+        {
+            code: "const JOE = 'schmoe';",
+            args: [1]
```

Comment:
Don't need `args` here.

You can use `ecmaFeatures` here with the latest checkin.